lib/fs/index.js

```diff
@@ -83,23 +84,23 @@ exports.read = function (fd, buffer, offset, length, position, callback) {
 // OR
 // fs.write(fd, string[, position[, encoding]], callback)
 // so we need to handle both cases
-exports.write = function (fd, buffer, a, b, c, callback) {
-  if (typeof arguments[arguments.length - 1] === 'function') {
-    return fs.write(fd, buffer, a, b, c, callback)
+exports.write = function (fd, buffer, ...args) {
+  if (typeof args[args.length - 1] === 'function') {
+    return fs.write(fd, buffer, ...args)
   }
 
   // Check for old, depricated fs.write(fd, string[, position[, encoding]], callback)
   if (typeof buffer === 'string') {
     return new Promise((resolve, reject) => {
```

Comment:
This here is actually code duplication now, and this `return` is not any different than to the other one. 

I think we can leave that `if`, because here it does not make the difference if there are two or three arguments. Thoughts?

Reply:
You're right; will fix.